File: ViewModel/Types/DefaultConfirmationMessageHandler.vb
Imports POFF.Kicker.ViewModel.Interfaces

Namespace Types

    Friend Class DefaultConfirmationMessageHandler
        Implements IConfirmationMessage

        Private Sub New()
            '
        End Sub

        Public Shared Empty As IConfirmationMessage = New DefaultConfirmationMessageHandler()

        Public Function Confirm(message As String) As Boolean Implements IConfirmationMessage.Confirm
            Return True
        End Function

    End Class

End Namespace

File: ViewModel/Types/DefaultConfirmationMessageHandler.cs
using POFF.Kicker.ViewModel.Interfaces;

namespace POFF.Kicker.ViewModel.Types
{

    internal class DefaultConfirmationMessageHandler : IConfirmationMessage
    {

        private DefaultConfirmationMessageHandler()
        {
            // 
        }

        public static IConfirmationMessage Empty = new DefaultConfirmationMessageHandler();

        public bool Confirm(string message)
        {
            return true;
        }

    }

}